test/packages/bad_deployment_mode_resources/manifest.yml

```diff
@@ -0,0 +1,58 @@
+format_version: 3.2.2
+name: bad_deployment_mode
+title: "Bad Deployment Mode"
+version: 0.0.1
+source:
+  license: "Apache-2.0"
+description: "This is a package that disables default policies before being supported."
+type: integration
+categories:
+  - custom
+conditions:
+  kibana:
+    version: "^8.12.2"
+  elastic:
+    subscription: "basic"
+screenshots:
+  - src: /img/sample-screenshot.png
+    title: Sample screenshot
+    size: 600x600
+    type: image/png
+icons:
+  - src: /img/sample-logo.svg
+    title: Sample logo
+    size: 32x32
+    type: image/svg+xml
+policy_templates:
+  - name: sample
+    title: Sample logs
+    description: Collect sample logs
+    deployment_modes:
+      default:
+        enabled: true
+      agentless:
+        enabled: true
+        organization: elastic
+        division: observability
+        team: obs-infraobs-integrations
+        resources:
+          memory: 1024M
+          cpu: "0.5"
+          disk: 10G
```

Comment:
Could we also add a "good" example [in the `good_v3` package](https://github.com/elastic/package-spec/blob/9655ea120e6afdc17b7dc41f9d652651b142e18e/test/packages/good_v3/manifest.yml#L37)?